### PR TITLE
Update Brief Summary of Data Fetching and Processing

### PR DESCRIPTION
**Pull request by bh2smith**
This was WAY out of date (only included steps to the holder fetching and processing).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,16 +50,28 @@ To, briefly summarize the approach we;
 4. Fetch Gnosis Chain Depositors
 5. Compute GNO holdings of LP providers based on share % in correspondence with the
    Pool's holdings (from step 1)
-6. Combine all of these values together to generate Mainnet and GnosisChain CSV
-   allocation files with columns
+6. Combine all of these values together to generate combined (Mainnet and GnosisChain
+   CSV) representing total GNO holdings across all eligible platforms.
+7. Fetch and Merge Trader Data from both networks
```
Comment (MartinquaXD) [inline README.md]: nit: captialization "Fetch and Merge Trader Data"

**Review by MartinquaXD — COMMENTED** [review]
Looks good. Just 1 nit

File: README.md (hunk 1 above)
Comment (bh2smith) [reply README.md]:
```suggestion
7. Fetch and merge trader data from both networks
```

File: README.md [hunk 2]
```diff
@@ -50,16 +50,28 @@ To, briefly summarize the approach we;
 4. Fetch Gnosis Chain Depositors
 5. Compute GNO holdings of LP providers based on share % in correspondence with the
    Pool's holdings (from step 1)
-6. Combine all of these values together to generate Mainnet and GnosisChain CSV
-   allocation files with columns
+6. Combine all of these values together to generate combined (Mainnet and GnosisChain
```
Comment (MartinquaXD) [inline README.md]: nit: sometimes it's GnosisChain sometimes Gnosis Chain.

File: README.md [hunk 3]
```diff
@@ -50,24 +50,36 @@ To, briefly summarize the approach we;
...
+The allocation files are committed to this repo and can be viewed here:
 
 - [Mainnet Allocations](allocations-mainnet.csv)
 - [Gnosis Chain Allocations](allocations-gchain.csv)
 
 # Allocations
 
 An Allocation (represented in the project as a `MerkleLeaf`), attributed to each
-eligible account, consists of several categories we briefly defined here (some of which
+eligible account, consists of several categories briefly define here (some of which
```
Comment (MartinquaXD) [inline README.md:82]: nit: define -> defined

**Review by MartinquaXD — APPROVED** [review]
LGTM. My nits have been addressed.

**Pull request merged**